### PR TITLE
New version: MongoDB.Server version 5.1.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34575)

### CODE REVIEW

File: manifests/m/MongoDB/Server/5.1.0/MongoDB.Server.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: MongoDB.Server
+PackageVersion: 5.1.0
+PackageLocale: en-US
+Publisher: MongoDB Inc.
+PublisherUrl: https://www.mongodb.com
+PublisherSupportUrl: https://support.mongodb.com/welcome
+PrivacyUrl: https://www.mongodb.com/legal/privacy-policy
+Author: MongoDB Inc.
+PackageName: MongoDB Server
```
Comment (OfficialEsco) [inline manifests/m/MongoDB/Server/5.1.0/MongoDB.Server.locale.en-US.yaml:12]:
```suggestion
PackageName: MongoDB
```

File: manifests/m/MongoDB/Server/5.1.0/MongoDB.Server.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ACCEPTEULA=1 ADDLOCAL=ServerService REBOOT=ReallySuppress
+UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/m/MongoDB/Server/5.1.0/MongoDB.Server.installer.yaml:17]:
```suggestion
UpgradeBehavior: uninstallPrevious
```

**Review by OfficialEsco — COMMENTED** [review]
Hmm, @jedieaston isn't this more correct?
image

**Comment by jedieaston** [comment]
Yes. The PackageName is a holdover from when we had two packages that used the same installer and ARP entry to signify the client vs the server.

**Review by jedieaston — APPROVED** [review]

**Pull request merged**